### PR TITLE
feat: polygon mana balance in the profile

### PR DESCRIPTION
**Pull request by AjimenezDCL**
#511 
Polygon MANA balance is now shown in the profile.

I'm not really happy with the implementation but the `ProfileHUD` needs quite the refactor. I tried to adjust to the current "paradigm" and created a ticket to solve it in the future: https://app.zenhub.com/workspaces/unity-6047bad476c3c0001942ee91/issues/decentraland/unity-renderer/549

### CODE REVIEW

File: unity-renderer/Assets/Scripts/MainScripts/DCL/ServiceProviders/TheGraph/TheGraph.cs [hunk 1]
```diff
@@ -90,6 +92,32 @@ public Promise<List<Land>> QueryLands(string tld, string address, float cacheMax
         return promise;
     }
 
+    public Promise<double> QueryMana(string address)
```
Comment (pbosio) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/ServiceProviders/TheGraph/TheGraph.cs:95]: Could we add something to explicitly point out that this is for matic/polygon chain?
Not sure what would be the best way to go, changing the method name or adding a new argument to the method like `bool usePolygonChain` and use one url or the other depending on that.

**Review by pbosio — APPROVED** [review]
LGTM

File: unity-renderer/Assets/Scripts/MainScripts/DCL/ServiceProviders/TheGraph/TheGraph.cs (hunk 1 above)
Comment (AjimenezDCL) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/ServiceProviders/TheGraph/TheGraph.cs:95]: I'm gonna rename it to `QueryPolygonMana` and once we port the retrieval of `MainNet/Ropsten` to Unity we can create another query (similar to how Lands are requested)

**Review by pravusjif — APPROVED** [review]

File: unity-renderer/Assets/Scripts/MainScripts/DCL/ServiceProviders/TheGraph/TheGraph.cs [hunk 2]
```diff
@@ -90,6 +92,32 @@ public Promise<List<Land>> QueryLands(string tld, string address, float cacheMax
...
+            .Then(resultJson =>
+            {
+                try
+                {
+                    JObject result = JObject.Parse(resultJson);
+                    JToken manaObject = result["data"]?["accounts"].First?["mana"];
+                    if (manaObject == null || !double.TryParse(manaObject.Value<string>(), out double parsedMana))
+                        throw new Exception($"QueryMana response couldn't be parsed: {resultJson}");
+
+                    promise.Resolve(parsedMana / 1e18);
```
Comment (BrianAmadori) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/ServiceProviders/TheGraph/TheGraph.cs:110]: Why are we dividing the value by `1e18`?

**Review by BrianAmadori — APPROVED** [review]

**Comment by AjimenezDCL** [comment]
> I'm approving this, BUT, there's no way of knowing what the icons stand for, and there's no good explanation in the tooltip:
> 
> image
> 
> I think we should hold this PR until we get a better tooltip or at least some way of improving the usability of this.

@BrianAmadori You are completely right about this, We are in the process of finding a more appropriated tooltip and hover text

**Review by kuruk-mm — APPROVED** [review]
Looks great

**Pull request merged**